src/metrics/pocket_area/voronoi_pocket_area.py

```diff
@@ -0,0 +1,34 @@
+from typing import Dict, List
+
+from scipy.spatial import Voronoi
+from shapely import Polygon
+
+from src.metrics.pocket_area.base import PocketArea, PocketAreaMetadata
+
+
+def voronoi_pocket_area(players: List[Dict]) -> PocketArea:
+
+    passer_idx = -1
+    passer_coord = (-1, -1)
+    for i, p in enumerate(players):
+        if p["role"] == "passer":
+            passer_idx = i
+            passer_coord = (p["x"], p["y"])
+            break
+
+    points = [(p["x"], p["y"]) for p in players]
```

Comment:
This might include players with other roles, like receivers, which we don't really need and might increase the time/memory cost of computing the voronoi cells, so let's only use passer, blockers, and rushers, which we get from the helper split function.